src/app/remotes/help-item-editor/help-item-editor.component.ts

```diff
@@ -48,7 +49,8 @@ import { HelpItemEditorDialogComponent } from './help-item-editor-dialog/help-it
   providers: [
     HelpsInternalAPIService,
     PortalMessageService,
-    providePortalDialogService(),
+    DialogService,
+    PortalDialogService,
```

Comment:
Do not import PrimeNG dialogService related services and use providePortalDialogService()

Reply:
Merge error. Fixed